store/project.js

```diff
@@ -102,11 +115,23 @@ export const mutations = {
       id: `row-${Date.now()}`,
       zones
     }
-    const rows = [...state.project.rows, row]
-    state.project = { ...state.project, rows }
+    const desktop = {
+      ...state.project.desktop,
+      rows: [...state.project.desktop.rows, row]
+    }
+    const mobile = {
+      ...state.project.mobile,
+      rows: [...state.project.mobile.rows, row]
+    }
+
+    state.project.currentStep === stepsEnum.desktop
+      ? (state.project = { ...state.project, desktop })
+      : (state.project = { ...state.project, mobile })
   },
   attachComponent(state, payload) {
-    const rows = state.project.rows.map((section) => {
+    const { currentStep } = state.project
+    let screenType = state.project[currentStep]
```

Comment:
well
let think about the next thing
what to do if we attach a component to desktop then shall we add it to mobile as well?